Lib/gftools/scripts/set_overlap_bits.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+from defcon import Font
+from pathops import Path
+from ufo2ft.filters.decomposeComponents import DecomposeComponentsFilter
+from ufo2ft.preProcessor import TTFPreProcessor
+import math
+from fontTools.designspaceLib import DesignSpaceDocument
+
+
+def set_overlap_bits(ufo):
+    # Skip setting bits if ufo
+    if any(g.lib.get("public.truetype.overlap") for g in ufo):
+        return
+    # Decompose components first because some component glyphs may have
+    # components that overlap each other
+    outline_glyphset = TTFPreProcessor(
+        ufo, filters=[DecomposeComponentsFilter()]
+    ).process()
+
+    overlaps = set()
+    for glyph in outline_glyphset.values():
+        skia_path = Path()
+        pen = skia_path.getPen()
+        for contour in glyph:
+            contour.draw(pen)
+        area = skia_path.area
+        # rm overlaps
+        skia_path.simplify()
+        simplified_area = skia_path.area
+        if not math.isclose(area, simplified_area, abs_tol=0.1):
```

Comment:
it's unusual to set abs_tol so much larger than rel_tol (default rel_tol is 1e-09), it'd take precedence most of the time, whereas in fact it's supposed to help with comparisons close 0 (where rel_tol tends to be too small), see https://docs.python.org/3/library/math.html#math.isclose
If you only care about comparing against an absolute value (and not do any relative comparisons) just do abs(a-b) <= abs_tol and you don't need to bother using isclose.

But actually, perhaps it would make more sense to use a tolernce that is relative to the font's unitsPerEm?
I don't know, something like `font.info.unitsPerEm / 10_000` (you'd still get 0.1 for fonts with 1000 UPEM, but if would increase/decrease accordingly)